mcp-server/tests/unit/tools/mock_operation.py

```diff
@@ -235,3 +235,23 @@ async def run_job(self, job_template_name: str, job_config: dict) -> str:
 
     async def cancel_job(self, job_id: str) -> str:
         return f"mock_job_cancelled: {job_id}"
+
+
+class MockStatisticOperation(StatisticOperation):
+    async def list_of_statistics(
+        self, metalake_name: str, metadata_type: str, metadata_fullname: str
+    ) -> str:
+        return "mock_statistics"
```

Comment:
Could you add the method argument  to the return value? so we could check argument too.